jobs/kpi-forecasting/kpi_forecasting/metric_hub.py

```diff
@@ -52,17 +59,32 @@ def __post_init__(self) -> None:
             "\n", "\n" + " " * 19
         )
 
+        # Add query snippets for segments
+        if self.segments:
+            segment_select_query = []
+            for alias, sql in self.segments.items():
+                segment_select_query.append(f"  {sql} AS {alias},")
+            self.segment_select_query = ",\n    ".join(segment_select_query)
+            self.segment_groupby_query = ",\n ,".join(self.segments.keys())
```

Comment:
Similar question here. Won't `",\n ,"` give you two commas in a row for every segment? So the groupby query would look like:
```sql
,
, test_segment1,
, test_segment2
```